src/FSharpVSPowerTools.Logic/VSUtils.fs

```diff
@@ -87,6 +87,20 @@ type ITextBuffer with
         let span = SnapshotSpan(x.CurrentSnapshot, 0, x.CurrentSnapshot.Length)
         event.Trigger(sender, SnapshotSpanEventArgs(span))
 
+type ITextView with
+    member x.GetCaretPosition () =
+        maybe {
+          let! point = x.TextBuffer.GetSnapshotPoint x.Caret.Position
+          let line = point.Snapshot.GetLineNumberFromPosition point.Position
+          let col = point.Position - point.GetContainingLine().Start.Position
+          return (line, col)
+        }
+    member x.posAtCaretPosition () =
```

Comment:
I guess I put it to avoid matching on GetSnapshotPoint.

Would you recommend something like this:
```fsharp
member x.GetCaretPosition () =
    match x.TextBuffer.GetSnapshotPoint x.Caret.Position with
    | None -> None
    | Some point ->
        let line = point.Snapshot.GetLineNumberFromPosition point.Position
        let col = point.Position - point.GetContainingLine().Start.Position
        Some (line, col)
```
or I'm missing something?